<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelalogin</class>
 <widget class="QMainWindow" name="janelalogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>janelalogin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background:#4169E1;  /* Gradiente de cinza para branco */
    border: none;  /* Remove a borda */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>411</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background: #FFFFFF;
    border: 1px solid #ccc;
    border-radius: 10px;
    padding: 10px;
    box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.4); /* Sombra forte */
}

QGroupBox:title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>289</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    QLineEdit {
       border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #d3d3d3;
      color: #333333;
   }
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>261</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    QLineEdit {
       border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #d3d3d3;
      color: #333333;
   }

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
       border: 1px solid #888888; 
       border-radius: 6px;
       padding: 5px;
       background-color: #4169E1;
       color: #FFFFFF;
	   font: bold;
   }

</string>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">z-index:1;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/user.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
